### ISSUE TITLE
Migration Question on Cancelation Survey

### ISSUE THREAD

**Issue by carinapilar** [Automattic/wp-calypso#60187]
### Quick summary

It showed a migration question on the cancellation survey, even though I didn't do any migration.

### Steps to reproduce

1. Start a new site, get a Biz plan, and AT the site.
2. Cancel the Biz plan, the site will be automatically reverted.
3. Get the Biz plan and AT the site again.
4. And cancel one more time.

### What you expected to happen

To not ask me about any migration.

### What actually happened

The cancellation survey had a migration question.

### Context

I found this when troubleshooting something else related to moving from AT to Simple a couple times. My guess is that during the revert/AT process a migration flag might be set, which causes the question to show up on the survey:

![CancelationSurvey](https://user-images.githubusercontent.com/16104522/149938553-a586f2e5-3078-4126-a991-70543e48cf45.jpg)

**Comment by Greatdane** [comment]
Hey @carinapilar I tried this a couple of times (AT, cancel, AT, cancel) but still got just these questions;

<img width="928" alt="Markup 2022-01-20 at 17 05 24" src="https://user-images.githubusercontent.com/5075840/150297919-8d0ebcf9-5531-4653-bdf6-3568e4946b34.png">

Can you reproduce it each time or is it possible something else on the site triggered the import? (I started with a fresh site)

**Comment by carinapilar** [comment]
Hey @Greatdane! I also started with a fresh site, I only ATed the site, installed a plugin (for additional Gutenberg blocks), canceled the site, which was reverted, got the plan one more time and canceled again, nothing related with any import.

I tried again now and wasn't able to replicate, this time though, when ATing the site again Jetpack got disconnected for some reason, which I'm not sure if it would impact the issue's reproducibility. 😕
